### DISCUSSION TITLE
Three ideas in one, but basically groups of tabs, autosort tabs by live, and edit color of tab by live

### DISCUSSION THREAD

**Discussion by nh-125** [Chatterino/chatterino2#3393] [Ideas]
I propose three new features,

Grouping (or bordering of user selected tabs) (potentially with a title as a text field for each)
Autosort live vs offline stream automatically into different sections, maybe just to the front/top like a stack/queue
Edit the background color of tab

**Comment by trollollol123** [comment]
I hope sorting tabs is a prio feature. This really is needed.

**Comment by Lopix77** [comment]
Refreshing this a little bit since it was reported but didn't get too much attention - I would specifically love to have Grouping for tabs. I am one of those people who adds tabs until I get blocked from connecting to more chats at Twitch, and sometimes it is hard to navigate tabs. 

I think this could become a popular feature and hopefully it wouldn't be a huge effort to implement - simply grouping tabs like you can in browsers and then naming the group (e.g. CS streamers / Dota streamers / forsen / variety streamers / offline chats / alts / VTubers etc.)

I use tabs as an indicator of whether someone is live more than follows so groups could have an indicator if any tab in the group is live.


There could also be something like "inactive tabs" which you can have removed from the main list so you don't actually connect to them but there is something like quickjoin as a bit of a workaround to the concurrent connections limit

**Comment by itsirfan05** [comment]
i hope this can still happen

**Reply by 8thony** [reply]
> Grouping (or bordering of user selected tabs) (potentially with a title as a text field for each)

- Probably related issue #2885

> Autosort live vs offline stream automatically into different sections, maybe just to the front/top like a stack/queue

- For now you can use the option "Tab visibility" and set it to "Only show live tabs" (Introduced in #4358)
	- For sorting tabs in the upcoming release (2.5.5) will be an option to manually sort tabs alphabetically (Introduced in #6551)
	- But for sorting Live Tabs there is still a feature request open #2810

> Edit the background color of tab

- Chatterino supports custom themes. Read more about this on the Wiki:
	- https://wiki.chatterino.com/Themes/
		- https://wiki.chatterino.com/Themes/#alternate-editors
